app/code/Magento/InventoryCatalog/Model/LocalCache/GetProductIdsBySkusCache.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Model\LocalCache;
+
+/**
+ * Local cache for service GetProductIdsBySkus.
+ */
+class GetProductIdsBySkusCache
```

Comment:
why do we need a separate class for such a straightforward operation? 
let's do that in the scope of classes where the cache is used.

And if you have CPD tests red because of copy-paste - skip that for one of the classes